service/matching/matcher_test.go

```diff
@@ -152,7 +152,7 @@ func (t *MatcherTestSuite) testRemoteSyncMatch(taskSource enumsspb.TaskSource) {
 		task, err := t.childMatcher.Poll(ctx, &pollMetadata{})
 		cancel()
 		if err == nil && !task.isStarted() {
-			task.finish(nil)
+			task.finish(nil, true)
```

Comment:
not this line, but line 167:
there's going to be another merge conflict with #6535, btw